### DISCUSSION TITLE
Gantt chart

### DISCUSSION THREAD

**Discussion by aperez900907** [mantinedev/mantine#1138] [Feature requests]
It's in the roadmap the implementation of the Gantt chart support?
will be very appreciated.
thanks

**Comment by rtivital** [comment]
No, this will not be added, use dedicated charts library.

**Comment by GZTimeWalker** [comment]
Hi @rtivital, Now that we have '@mantine/charts', are we reconsidering adding the gantt chart feature?

Something like notion's timeline view, a scheduling gantt chart, which shows the task duration in it?
